core/services/keystore/eth_test.go

```diff
@@ -34,6 +33,7 @@ func Test_EthKeyStore(t *testing.T) {
 		require.NoError(t, utils.JustError(db.Exec("DELETE FROM eth_key_states")))
 		keyStore.Unlock(cltest.Password)
 	}
+	stateTableName := ethkey.State{}.TableName()
```

Comment:
I'd say its reasonable to just use a hardcoded string for table names

Reply:
I just wanted to avoid copying table names here, but I see your point, will use just strings.